content/news/releases/prestashop-8-2-1-maintenance-release.md

```diff
@@ -0,0 +1,84 @@
+
+![PrestaShop 8.2.1 is available!](/assets/images/2024/09/banner_8.2_1534x424.jpg)
+
+<hr>
+
+We are excited to announce the release of PrestaShop 8.2.1, a new patch that not only addresses [a significant number of bugs](https://github.com/PrestaShop/PrestaShop/pulls?q=is%3Apr+is%3Amerged+milestone%3A8.2.1+label%3A%22Bug+fix%22+-label%3A%22E2E+Tests%22+) reported by our dedicated community and Quality Assurance team but also introduces a valuable new feature. This feature allows module developers to filter the display of other modules in PrestaShop, helping them avoid the use of overrides.
+
+By reducing the reliance on overrides, this enhancement improves the overall stability and compatibility of the platform. We believe these improvements will contribute to a better and more efficient PrestaShop experience for everyone.
+
+<hr>
+
+## What's new in PrestaShop 8.2.1?
+
+**Filtering of the modules in the front office**
+
+The method `Hook::getHookModuleExecList` [is now customizable by module developers](https://github.com/PrestaShop/PrestaShop/pull/37125). This method, which filters the list of modules executed for a given hook, normally filtered based on the hook's name. With the new update, developers can customize this filtering process using other criteria. This addition is particularly beneficial for module developers aiming to minimize the use of overrides, especially for modules that filter the list of executed modules for specific hooks (e.g., solutions for handling Consent Mode and privacy solutions).
+
+Typically, we do not introduce such new features in patch versions, as they are primarily intended for bug fixes and minor improvements. However, the significance of this new feature has led us to make an exception.
+
+You can find more information about this new feature in the [pull request](https://github.com/PrestaShop/PrestaShop/pull/37125/), and download the example module from the [PrestaShop Example Modules GitHub repository](https://github.com/PrestaShop/example-modules/demofiltermodules). We believe that including this feature in the patch version will help module developers enhance their modules and ensure better compatibility with future versions of PrestaShop by reducing the reliance on overrides.
+
+PS: It is worth mentioning that it is possible to filter the back office modules as well. The use case is not as common as for the front office, but it is possible.
+
+**Better way to handle errors when installing a module**
+
+In [this pull request](https://github.com/PrestaShop/PrestaShop/pull/36612/) we improved the way we handle errors when installing a module. Previously, it was not possible to prevent the installation of a module even if certain essential criteria were not met in the `actionModuleInstallBefore` hook, which could lead to issues with, for example, required modules.
```

Comment:
```suggestion
In [this pull request](https://github.com/PrestaShop/PrestaShop/pull/36612/), we improved the way we handle errors when installing a module. Previously, it was not possible to prevent the installation of a module even if certain essential criteria were not met in the `actionModuleInstallBefore` hook, which could lead to issues with, for example, required modules.
```